labels.mss

```diff
@@ -129,166 +129,103 @@
   [highway = 'motorway_trunk'][zoom >= 10],
   [highway = 'primary'][zoom >= 11],
   [highway = 'secondary'][zoom >= 12] {
-    shield-name: "[refs]";
-    shield-size: @shield-size;
-    shield-line-spacing: @shield-line-spacing;
-    shield-placement: line;
-    shield-spacing: @shield-spacing;
-    shield-repeat-distance: @shield-repeat-distance;
-    shield-margin: @shield-margin;
-    shield-face-name: @sans;
-    shield-clip: @shield-clip;
-
-    [highway = 'motorway_trunk'] {
-      shield-fill: @motorway-trunk-shield-fill;
-      shield-file: url("symbols/shields/motorway-trunk_[width]x[height].svg");
-    }
-
-    [highway = 'primary'] {
-      shield-fill: @primary-shield-fill;
-      shield-file: url("symbols/shields/primary_[width]x[height].svg");
-    }
-
-    [highway = 'secondary'] {
-      shield-fill: @secondary-shield-fill;
-      shield-file: url("symbols/shields/secondary_[width]x[height].svg");
-    }
+      text-name: "[refs]";
+      text-size: 7;
+      text-fill: @road_text;
+      text-spacing: 600;
+      text-clip: false;
+      text-placement: line;
+      text-face-name: @sans;
+      text-repeat-distance: @major-highway-text-repeat-distance;
+      text-halo-radius: @standard-halo-radius;
   }
 }
 
 #roads-text-ref {
-  [highway = 'motorway_trunk'],
```

Comment:
The two of them were merged https://github.com/cyclosm/cyclosm-cartocss-style/blob/master/project.mml#L2011 in SQL queries. We should either keep `motorway_trunk` or drop the merge in `roads-text-ref` SQL.

Reply:
I don’t remind to have done that change, must come from the copy past I have done to write that layer from name layer.